### DISCUSSION TITLE
How can I disable the juggernaut model installation on every startup?

### DISCUSSION THREAD

**Discussion by DeVilelo** [lllyasviel/Fooocus#2715] [Q&A] [answered]
I have found several models that are enough for all the tasks I need, the juggernaut model seems to me no longer needed, but after deleting yhe model fooocus downloads the juggernaut again, it would not make sense, but I have limited memory, thank you in advance for your help

**Comment by mashb1t** [comment, marked as answer]
see https://github.com/lllyasviel/Fooocus/discussions/2681 and many previous discussions ;)
